nautobot/wireless/templates/wireless/radioprofile_retrieve.html

```diff
@@ -0,0 +1,93 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+
+{% block content_left_page %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Radio Profile</strong>
+        </div>
+        <table class="table table-hover panel-body attr-table">
+            <tr>
+                <td>Name</td>
+                <td>
+                    {{ object.name }}
+                </td>
+            </tr>
+            <tr>
+                <td>Description</td>
+                <td>{{ object.description|placeholder }}</td>
+            </tr>
+            <tr>
+                <td>Frequency</td>
+                <td>{{ object.frequency }}</td>
+            </tr>
+            <tr>
+                <td>Channel Width</td>
+                <td>
+                    {% if object.channel_width %}
+                        {% for width in object.channel_width %}
+                            <span class="label label-default">{{ width }}MHz</span>
+                        {% endfor %}
+                    {% else %}
+                        {{ None|placeholder }}
+                    {% endif %}
+                </td>
+            </tr>
+            <tr>
+                <td>Allowed Channel List</td>
+                <td>
+                    {% if object.allowed_channel_list %}
+                        {% for channel in object.allowed_channel_list %}
+                            <span class="label label-default">{{ channel }}</span>
+                        {% endfor %}
+                    {% else %}
+                        {{ None|placeholder }}
+                    {% endif %}
+                </td>
+            </tr>
+            <tr>
+                <td>Transmit Power Minimum</td>
+                <td>
+                    {% if object.tx_power_min %}
+                        {{ object.tx_power_min }} dBm
+                    {% else %}
+                        {{ None|placeholder }}
+                    {% endif %}
```

Comment:
Might make sense to create a template tag for this since you're repeating it at least three times?

Reply:
Sure!